File: PY/find_emp.py
def salary_analysis(salaries):

    highest = max(salaries)
    lowest = min(salaries)
    average = sum(salaries) / len(salaries)

    print("Highest:", highest)
    print("Lowest :", lowest)
    print("Average:", average)

salary_analysis([25000,30000,45000,28000])

File: JAVA/find_emp.java
public class find_emp {

    public static void main(String[] args) {

        int[] salary = {25000,30000,45000,28000};

        int highest = salary[0];
        int lowest = salary[0];
        int sum = 0;

        for(int s : salary){

            if(s > highest)
                highest = s;

            if(s < lowest)
                lowest = s;

            sum += s;
        }

        double avg = (double)sum / salary.length;

        System.out.println("Highest : " + highest);
        System.out.println("Lowest : " + lowest);
        System.out.println("Average : " + avg);

    }
}